mmv1/templates/terraform/examples/tpu_v2_vm_v5p.tf.erb

```diff
@@ -0,0 +1,21 @@
+resource "google_tpu_v2_vm" "<%= ctx[:primary_resource_id] %>" {
+  provider = google-beta
+
+  name = "<%= ctx[:vars]["vm_name"] %>"
+  zone = "us-east5-a"
```

Comment:
At this time, it appears that it is only available the zone for me due to quota restriction.